modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -223,6 +223,77 @@ validateMetaData <- function(metaData, configList, username, formatSettings = li
 
   return(list(validatedMetaData=validatedMetaData, duplicateExperimentNamesAllowed=duplicateExperimentNamesAllowed, useExisting=useExisting))
 }
+
+
+saveIncomingEndpointData <- function(codes, codeKind) {
```

Comment:
nit: this function name is pretty broad. Something narrower like `saveEndpointDDictValues` or `saveEndpointCodeTables` might be clearer as to what's in and out of scope of this function.

Reply:
Makes sense!